Remotion/Mixins/Core/Context/Serialization/AttributeMixinContextOriginSerializer.cs

```diff
@@ -24,10 +24,10 @@ namespace Remotion.Mixins.Context.Serialization
   /// </summary>
   public class AttributeMixinContextOriginSerializer : ArrayMixinContextOriginSerializer
   {
-    protected override object? ConvertToStorageFormat<T> (T value)
+    protected override object ConvertToStorageFormat<T> (T value)
     {
```

Comment:
Shouldn't `T` be constrained as notnull? See remaining files in this inheritance hierharchy

Reply:
Constraints are inherited from the base method, `T` is classified `notnull` here.

Reply:
oh, okay. wonderful consistency there, C#.